examples/umbrella-sampling-pmf/umbrella-sampling.py

```diff
@@ -1,170 +1,196 @@
-# Example illustrating the application of MBAR to compute a 1D PMF from an umbrella sampling simulation.
-#
-# The data represents an umbrella sampling simulation for the chi torsion of a valine sidechain in lysozyme L99A with benzene bound in the cavity.
-# 
-# REFERENCE
-# 
-# D. L. Mobley, A. P. Graves, J. D. Chodera, A. C. McReynolds, B. K. Shoichet and K. A. Dill, "Predicting absolute ligand binding free energies to a simple model site," Journal of Molecular Biology 371(4):1118-1134 (2007).
-# http://dx.doi.org/10.1016/j.jmb.2007.06.002
+"""
+Example illustrating the application of MBAR to compute a 1D PMF from an umbrella sampling simulation.
+
+The data represents an umbrella sampling simulation for the chi torsion of
+a valine sidechain in lysozyme L99A with benzene bound in the cavity.
+
+Reference:
+
+    D. L. Mobley, A. P. Graves, J. D. Chodera, A. C. McReynolds, B. K. Shoichet and K. A. Dill,
+    "Predicting absolute ligand binding free energies to a simple model site,"
+    Journal of Molecular Biology 371(4):1118-1134 (2007).
+    http://dx.doi.org/10.1016/j.jmb.2007.06.002
+
+
+TODO
+----
+
+There are several undefined names!
+
+* `ntot`
+* `bin_edges`
+* `chi_n`
+* `delta`
```

Comment:
@mrshirts These names are undefined in the script, and I don't know what they mean :/

Reply:
I think these should be fixed if I read this correctly?